robot/impl/resource_manager.go

```diff
@@ -377,6 +377,11 @@ func (manager *resourceManager) completeConfig(
 		}
 		manager.logger.Debugw("we are now handling the resource", "resource", r)
 		if c, ok := wrap.config.(config.Component); ok {
+			_, err := c.Validate("")
+			if err != nil {
```

Comment:
should wrap the errors with something that indicates that the config is broken and what the resource name is

Reply:
I added an error wrap msg and things started to look a bit redundant. for example 

**without the added wrap msg**: 'resource "rdk:component:base/" not available: Config validation error found in component: : error validating "": "name" is required'

**With added wrap msg**: 'resource "rdk:component:base/" not available: Config validation error found in component: : error validating "": "name" is required'

Reply:
these look the same?

Reply:
Ignore the previous msg, the logs look good